src/EnergyPlus/UnitarySystem.cc

```diff
@@ -2251,14 +2273,15 @@ namespace UnitarySystems {
                 this->m_MSCoolingSpeedRatio[Iter] = this->m_CoolVolumeFlowRate[Iter] / this->m_DesignFanVolFlowRate;
             }
 
-            if (MSHPIndex > 0) {
-                this->m_IdleVolumeAirRate = this->m_MaxCoolAirVolFlow * state.dataUnitarySystems->designSpecMSHP[MSHPIndex].noLoadAirFlowRateRatio;
-                this->m_IdleMassFlowRate = this->m_IdleVolumeAirRate * state.dataEnvrn->StdRhoAir;
-                this->m_IdleSpeedRatio = this->m_IdleVolumeAirRate / this->m_DesignFanVolFlowRate;
+            if (MSHPIndex > -1) {
```

Comment:
Note the change from 0 to -1 here. That was a mistake from the get go. designSpecMSHP is 0 based.